lib/rubocop/cop/rails/content_tag.rb

```diff
@@ -3,30 +3,29 @@
 module RuboCop
   module Cop
     module Rails
-      # This cop checks that `tag` is used instead of `content_tag`
-      # because `content_tag` is legacy syntax.
+      # This cop checks legacy syntax usage of `tag`
       #
-      # NOTE: Allow `content_tag` when the first argument is a variable because
-      #      `content_tag(name)` is simpler rather than `tag.public_send(name)`.
+      # NOTE: Allow `tag` when the first argument is a variable because
+      #      `tag(name)` is simpler rather than `tag.public_send(name)`.
       #
       # @example
       #  # bad
-      #  content_tag(:p, 'Hello world!')
-      #  content_tag(:br)
+      #  tag(:p)
+      #  tag(:br, {class: 'classname'})
```

Comment:
Can you omit the redundant parentheses?
```suggestion
      #  tag(:br, class: 'classname')
```